navbar-service.js

```diff
@@ -28,6 +28,15 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  // service.display may be a string or an array of strings
+  // valid values: all, brand, menu, toolbar
+  service.display = 'all';
+  service.siteTitle = null;
+
+  $rootScope.$on("$routeChangeSuccess", function() {
```

Comment:
What's this event handler about? Does this mean that, in an application that always wants a particular display setting, it will need every route to set that setting?